docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    pdf = pd.DataFrame({
</pre><p>Let's use a variable (<code>num_rows</code>) to store  <code>300_000_000.</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done